### PR TITLE
chore: add missing tf variables required to make fury work on gke using kube 1.25

### PR DESCRIPTION
**Pull request by omissis**

### CODE REVIEW

**Comment by omissis** [comment]
Should be merged after the following two PRs are merged https://github.com/sighupio/fury-gke-installer/pull/32 and https://github.com/sighupio/fury-gke-installer/pull/33 and gke installer v1.10.0 is released

File: data/provisioners/cluster/gke/main.tf [hunk 1]
```diff
@@ -17,8 +17,13 @@ terraform {
...
 }
 
+provider "google" {
+  project     = var.provider_project
+  region      = var.provider_region
+}
+
 module "fury" {
-  source = "github.com/sighupio/fury-gke-installer//modules/gke?ref=v1.9.0"
+  source = "github.com/sighupio/fury-gke-installer//modules/vpc-and-vpn?ref=v1.10.0"
```
[the remark this answers is not in the document]
Comment (omissis) [reply data/provisioners/cluster/gke/main.tf:26]: right, I copypasted wrong

File: internal/bootstrap/provisioners/gcp/provisioner.go [hunk 2]
```diff
@@ -156,6 +157,9 @@ func (d GCP) createVarFile() (err error) {
 	var buffer bytes.Buffer
 	spec := d.config.Spec.(cfg.GCP)
 
+	buffer.WriteString(fmt.Sprintf("provider_region = \"%v\"\n", spec.Region))
+	buffer.WriteString(fmt.Sprintf("provider_project = \"%v\"\n", spec.Project))
```
[the remark this answers is not in the document]
Comment (omissis) [reply internal/bootstrap/provisioners/gcp/provisioner.go:161]: we could, but terraform would probably complain anyways soon after this

File: automated-tests/e2e-tests/gcp-gke/cluster.tpl.yml [hunk 3]
```diff
@@ -6,15 +6,13 @@ kind: Cluster
 metadata:
   name: e2e-${CI_BUILD_NUMBER}
 spec:
-  version: 1.24.5-gke.600
+  version: 1.25
```
[the remark this answers is not in the document]
Comment (omissis) [reply automated-tests/e2e-tests/gcp-gke/cluster.tpl.yml:9]: it did work in my tests, with both 1.23 and 1.25

[the remark this answers is not in the document]
Comment (ralgozino) [reply automated-tests/e2e-tests/gcp-gke/cluster.tpl.yml:9]: if it works, then great! maybe older versions of the terraform provider required setting the full name 🤷

File: internal/bootstrap/provisioners/gcp/provisioner.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (ralgozino) [reply internal/bootstrap/provisioners/gcp/provisioner.go:161]: if these 2 parameters are mandatory we can leave it as-is

**Review by ralgozino — APPROVED** [review]

**Pull request merged**